client/src/views/organizations/creation/CreateOrganizationSaas.vue

```diff
@@ -138,12 +142,25 @@ const currentError = ref<Translatable | undefined>(undefined);
 const saveStrategy = ref<DeviceSaveStrategy | undefined>(undefined);
 const availableDevice = ref<AvailableDevice | undefined>(undefined);
 const sequesterKey = ref<string | undefined>(undefined);
+const serverConfig = ref<ServerConfig | undefined>(undefined);
 
 onMounted(async () => {
+  let server!: string;
```

Comment:
You could declare `let server: string = Env.getSaasServers()[0] ?? '';` and remove the `else` repeated twice below